specification/billing/resource-manager/Microsoft.Billing/preview/2024-08-02-preview/migration.json

```diff
@@ -0,0 +1,204 @@
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration mapping details.",
+            "schema": {
+              "$ref": "#/definitions/MigrationMappingResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "MigrationDetails": {
+            "$ref": "./examples/MigrationDetails.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Migrates from source to destination billing scope",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "Migration_Put",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": ".,/types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "parameters",
+            "description": "Migration mapping details.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrationMappingProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration details.",
+            "schema": {
+              "$ref": "#/definitions/MigrationResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "MigrationDetails": {
+            "$ref": "./examples/MigrationDetails.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MigrationMappingProperties": {
+      "description": "Migration mapping properties.",
+      "type": "object",
+      "properties": {
+        "destinationBillingScope": {
+          "description": "Destination billing scope to be mapped.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Migration Request status.",
+          "type": "string"
+        }
+      }
+    },
+    "MigrationResponse": {
+      "description": "Response contains Migration details.",
+      "type": "object",
+       "properties": {
+        "id": {
+          "description": "Resource id",
+          "readOnly": true,
+          "type": "string"
+        },
```

Comment:
If this is a fully qualified resource ID then please specify "arm-id" format.

Reply:
If this is a fully qualified resource ID then please specify "arm-id" format.

Reply:
Delete file from PR